src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const li = document.querySelectorAll('li');
+
+li.forEach((element) => {
+  if (element.childNodes.length > 1) {
+    const spanWrapper = document.createElement('span');
+
+    spanWrapper.innerHTML = `${element.firstChild.textContent}`;
```

Comment:
The use of `element.firstChild.textContent` assumes that the first child of the `li` element is a text node. If the first child is not a text node, this will not work as expected. Consider checking the node type or using a more robust method to get the text content.